client/app/cloud/project/compute/infrastructure/image.service.js

```diff
@@ -60,6 +60,57 @@
             }
             return augmentedImage;
         }
+
+        static getImageTypes (images) {
+            return _.uniq(_.pluck(images, "type"));
```

Comment:
_.pluck is deprecated in Lodash4. You can, instead, use:
_.uniq(_.map(images, "type")

Reply:
Done